src/main/resources/data/computercraft/lua/bios.lua

```diff
@@ -954,6 +964,11 @@ settings.define("lua.function_source", {
     description = "Show where a function was defined when printing functions.",
     type = "boolean",
 })
+settings.define("bios.strict_globals", {
+    default = false,
+    description = "Prevents assigning variables into a program's enviroment. Make sure you use the local keyword or assign to _G explicitly.",
```

Comment:
```suggestion
    description = "Prevents assigning variables into a program's environment. Make sure you use the local keyword or assign to _G explicitly.",
```

Sorry, should have spotted this before.